src/settings/ubuntuSettings.json

```diff
@@ -58,6 +58,12 @@
       "metadata": {
         "description": "Unique identifiers to allow the Azure Infrastructure to understand the origin of resources deployed to Azure. You do not need to supply a value for this."
       }
+    },
+    "dataDogAPIKey":{
```

Comment:
So based off this documentation (https://docs.microsoft.com/en-us/powershell/module/azurerm.resources/new-azurermresourcegroupdeployment?view=azurermps-6.11.0) it seems like we can just pass the API Key param using the `TemplateParameterFile` as a string which will then get used downstream